catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -144,7 +171,7 @@
  * services.
  */
 public abstract class AbstractCswSource extends MaskableImpl
-        implements FederatedSource, ConnectedSource, ConfiguredService {
+        implements CatalogStore, FederatedSource, ConnectedSource, ConfiguredService {
```

Comment:
The plan was that other thing would use catalog store in the future. For now I went ahead and moved the catalog store implementation out of AbstractCswSource and into its own class AbstractCswStore.